components/RefinementList.js

```diff
@@ -108,10 +90,9 @@ RefinementList.propTypes = {
 
 RefinementList.defaultProps = {
   cssClasses: {
-    root: null,
     item: null,
     list: null
   }
 };
 
-module.exports = autoHide(RefinementList);
+module.exports = autoHide(headerFooter(RefinementList));
```

Comment:
What about we export the components pure and maybe compose all the features in the widgets?